lib/rules.js

```diff
@@ -222,7 +222,16 @@ var rules = {
   'isURL': {
     fn: function(x, opt) {
       if (typeof x !== 'string') { throw new Error ('Value was not a string.'); }
-      return validator.isURL(x, opt === true ? undefined : opt);
+      var defaultOptions = {};
+      // As of Validator 8.0.0, isURL requires `require_tld` to be set to false to validate localhost as a URL.
+      if(x.match(/^(https?:\/\/)?localhost(\:|$)/)) {
```

Comment:
What about `localhost/foo' with no colon?
Also, should protocol be required?  What about other protocols like `file://` or `slack://`?

By “should”, I mean so that it maintains compatibility with the previous version